queries.yaml

```diff
@@ -112,6 +112,7 @@ layers:
     transform:
       - TileStache.Goodies.VecTiles.transform.tags_create_dict
       - TileStache.Goodies.VecTiles.transform.add_iata_code_to_airports
+      - TileStache.Goodies.VecTiles.transform.normalize_leisure_kind
```

Comment:
Depends on whether we want to write another function the next time we normalise another leisure kind or extend the existing one. I don't have a strong feeling either way - do you prefer `normalize_fitness_kind`?